trie/proof.go

```diff
@@ -35,9 +36,12 @@ import (
 // with the node that proves the absence of the key.
 func (t *Trie) Prove(key []byte, fromLevel uint, proofDb ethdb.KeyValueWriter) error {
 	// Collect all nodes on the path to key.
+	var (
+		prefix []byte
```

Comment:
This prefix is just a leak from a followup PR, right?

Reply:
Prefix is used for resolving node.